zebra-consensus/src/checkpoint.rs

```diff
@@ -412,6 +412,18 @@ impl CheckpointVerifier {
             .entry(height)
             .or_insert_with(|| QueuedBlockList::with_capacity(1));
 
+        let hash = block.as_ref().into();
```

Comment:
We have a convenience method now:
```suggestion
        let hash = block.hash();
```

Reply:
:+1:, will add a fixup.